modules/ROOT/pages/policy-mule4-jwt-validation.adoc

```diff
@@ -70,6 +70,12 @@ The JWT is also propagated downstream, and you can access it by using the follow
 
 `#[authentication.properties.jwt]`
 
+=== Retry Mechanism
+
+If you selected JSON Web Key Sets (JWKS) as JWT Key Origin, the policy uses a retry mechanism to handle failed JWKS fetches. Every time the cached JWKS expires or is absent and a new request is initiated, a request is sent to the JWKS service. If the request fails, a sequence of asynchronous requests with an exponential backoff delay are sent to the JWKS service until a new JWKS is obtained and cached again. 
+
+Until the recaching completes, the expired JWKS is used to process the request, and a `503` error is returned if no request is found. This enables you to use a cached value in upcoming requests, even if the previous requests failed.
```

Comment:
see style ref. entry for "complete":
```suggestion
Until the recaching is complete, the expired JWKS is used to process the request, and a `503` error is returned if no request is found. This enables you to use a cached value in upcoming requests, even if the previous requests failed.
```
or
```suggestion
Until the recaching finishes, the expired JWKS is used to process the request, and a `503` error is returned if no request is found. This enables you to use a cached value in upcoming requests, even if the previous requests failed.
```

Reply:
Can you clarify to what "This" refers in "This enables you...": is it the fact that the expired JWKS is used, or that the error is returned, or both?

Reply:
Done! Thanks!